### ISSUE TITLE
Don’t assign Unicodes to glyphs with suffixes

### ISSUE THREAD

**Issue by rbmntjs** [jenskutilek/RFUnicodeInfo#3]
Expected behaviour of the ‘Assign All’ function is that it only applies unambiguous Unicode values. Actual behaviour is that it will assign Unicodes to suffixed glyphs as well, such as for example `one.sup` getting assigned `U+0031` – it should never assume that suffixed glyphs have a Unicode.

**Comment by jenskutilek** [comment]
Hm, I can't see this problem ... what's your use case? You have a UFO containing `one.sup` withouth Unicode and it gets a value assigned? Does `one` exist in the UFO as well?

**Comment by rbmntjs** [comment]
Yes and yes. This is in one of the 3.3 betas – I can test it in the latest with a clean file to see if it still happens.
